ckanext/blob_storage/tests/test_blob_storage_bug_fix.py

```diff
@@ -0,0 +1,43 @@
+"""Tests for plugin.py."""
+import logging
+import mock
+import pytest
+
+# encoding: utf-8
```

Comment:
This line was needed for defining encoding in early python2 days. Right now since py3 utf-8 is the default source encoding. You don't need this (also when it was needed it was at the top of the file)